lib/t/news.ex

```diff
@@ -14,12 +14,177 @@ defmodule T.News do
+              "question" => "instagram",
+              "rotation" => -17
+            }
+          ],
+          "size" => [375, 667]
+        },
+        %{
+          "background" => %{"color" => "#F97EB9"},
+          "labels" => [
+            %{
+              "zoom" => 1.0,
+              "value" =>
+                dgettext(
+                  "news",
+                  "Теперь стикеры-контакты -\nединственный способ\nкоммуникации."
+                ),
+              "position" => [24.0, 200.0],
+              "corner_radius" => 0,
+              "alignment" => 0
+            },
+            %{
+              "zoom" => 1.0,
+              "value" => dgettext("news", "Общайся с мэтчами,\nгде тебе удобно."),
+              "position" => [24.0, 319.0],
+              "corner_radius" => 0,
+              "alignment" => 0
+            },
+            %{
+              "zoom" => 1.0,
+              "value" =>
+                dgettext(
+                  "news",
+                  "Это ещё не всё 👉"
+                ),
+              "position" => [150.0, 500.0],
+              "corner_radius" => 0,
+              "alignment" => 0
+            }
+          ],
+          "size" => [375, 667]
+        },
+        %{
+          "background" => %{"color" => "#F97EB9"},
+          "labels" => [
+            %{
+              "zoom" => 1.0,
+              "value" => dgettext("news", "Приватная страница 👀"),
+              "position" => [24.0, 131.0],
+              "corner_radius" => 0,
+              "alignment" => 0
+            },
+            %{
+              "zoom" => 1.0,
+              "value" =>
+                dgettext(
+                  "news",
+                  "Создай приватную страницу,\nона будет видна только\nтвоим мэтчам."
+                ),
+              "position" => [24.0, 200.0],
+              "corner_radius" => 0,
+              "alignment" => 0
+            },
+            %{
+              "zoom" => 1.0,
+              "value" =>
+                dgettext(
+                  "news",
+                  "На нее можно поместить\nсвой контакт и что-то\nболее личное про тебя."
+                ),
+              "position" => [24.0, 322.0],
+              "corner_radius" => 0,
+              "alignment" => 0
+            },
+            %{
+              "zoom" => 1.0,
+              "value" =>
+                dgettext(
+                  "news",
+                  "Выглядит вот так 👉"
+                ),
+              "position" => [130.0, 500.0],
+              "corner_radius" => 0,
+              "alignment" => 0
+            }
+          ],
+          "size" => [375, 667]
+        },
+        %{
+          "blurred" => %{
+            "proxy" =>
+              "https://d3r9yicn85nax9.cloudfront.net/Phqo20PDBNkUCjsdts6lDMc5JD_TxhpXtlCaB6bp3LM/fit/1200/0/sm/0/aHR0cHM6Ly9zaW5jZS13aGVuLWFyZS15b3UtaGFwcHkuczMuYW1hem9uYXdzLmNvbS81Y2ZiZTk2Yy1lNDU2LTQzYmItOGQzYS05OGU4NDljMDBkNWQ",
```

Comment:
should be dynamic

Reply:
should it use sth like this: https://github.com/getsince/test3/blob/master/lib/t_web/views/view_helpers.ex#L60?

Reply:
yes